### PR TITLE
Add CognitiveServices.ComputerVision tests for full coverage

### PR DESCRIPTION
**Pull request by cthrash**
A number of bugs were discovered in the Swagger, and were fixed in this [PR](https://github.com/Azure/azure-rest-api-specs/pull/3145).

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [X] Please add REST spec PR link to the SDK PR
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/_metadata/cognitiveservices_data-plane_ComputerVision.txt [hunk 1]
```diff
@@ -1,11 +1,11 @@
-2018-01-22 18:37:37 UTC
+2018-05-25 15:48:30 UTC
 
 1) azure-rest-api-specs repository information
-GitHub user: Azure
-Branch:      current
-Commit:      cb9c18b988dd67894653a34dc55b7978403b120a
+GitHub user: cthrash
```
Comment (dsgouda) [inline src/SDKs/_metadata/cognitiveservices_data-plane_ComputerVision.txt]: Please generate code from the Azure master branch, we cannot accept PRs with code generated from non-master branches

File: src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision.Tests/Microsoft.Azure.CognitiveServices.Vision.ComputerVision.Tests.csproj [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>Microsoft.Azure.CognitiveServices.Vision.ComputerVision.Tests Class Library</Description>
     <AssemblyName>Microsoft.Azure.CognitiveServices.Vision.ComputerVision.Tests</AssemblyName>
-    <VersionPrefix>1.0.2-preview</VersionPrefix>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision.Tests/Microsoft.Azure.CognitiveServices.Vision.ComputerVision.Tests.csproj]: The test project is never published, we recommend setting the version to `1.0.0` and never change it.

File: src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Microsoft.Azure.CognitiveServices.Vision.ComputerVision.csproj [hunk 3]
```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Vision.ComputerVision</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services ComputerVision APIs.</Description>
-    <VersionPrefix>1.0.2-preview</VersionPrefix>
+    <VersionPrefix>1.0.3</VersionPrefix>
     <AssemblyName>Microsoft.Azure.CognitiveServices.Vision.ComputerVision</AssemblyName>
     <PackageTags>Microsoft Cognitive Services;Cognitive Services;Cognitive Services SDK;REST HTTP client;Computer Vision;Computer Vision API;Computer Vision SDK;Vision;netcore451511</PackageTags>
     <PackageReleaseNotes>This is a preview release of the Cognitive Services Computer Vision SDK.</PackageReleaseNotes>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Microsoft.Azure.CognitiveServices.Vision.ComputerVision.csproj]: Please update PackageReleaseNotes

**Review by dsgouda — CHANGES_REQUESTED** [review]

**Comment by dsgouda** [comment]
Also, please rebase the commits to a single commit and update the PR

File: src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision.Tests/Microsoft.Azure.CognitiveServices.Vision.ComputerVision.Tests.csproj (hunk 2 above)
Comment (cthrash) [reply src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision.Tests/Microsoft.Azure.CognitiveServices.Vision.ComputerVision.Tests.csproj]: Done

**Review by cthrash — COMMENTED** [review]
Updates made based on comments

**Comment by cthrash** [comment]
bump

File: src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Microsoft.Azure.CognitiveServices.Vision.ComputerVision.csproj [hunk 4]
```diff
@@ -6,10 +6,10 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Vision.ComputerVision</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services ComputerVision APIs.</Description>
-    <VersionPrefix>1.0.2-preview</VersionPrefix>
+    <VersionPrefix>1.0.3</VersionPrefix>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Microsoft.Azure.CognitiveServices.Vision.ComputerVision.csproj]: Please confirm you plan on releasing a stable version here. If yes, versioning should start from `1.1.0`
Also, all the projects need an `AssemblyInfo.cs` file similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/Properties/AssemblyInfo.cs) with appropriate package versions.

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good for the most part, needs a couple of comments addressed. Apologize for the turnaround

**Comment by cthrash** [comment]
Thanks for the review.  Version updated, metadata added.

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
Will merge on CIs passing

**Pull request merged**